lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -63,15 +67,18 @@ export class AuthorizationCodeClient extends BaseClient {
      * @param request
      */
     async acquireToken(request: CommonAuthorizationCodeRequest, authCodePayload?: AuthorizationCodePayload): Promise<AuthenticationResult> {
+        if (!request || StringUtils.isEmpty(request.code)) {
```

Comment:
nit: (never been a fan of this "helper")
```suggestion
        if (!request || !request.code) {
```